packages/flame/lib/src/device.dart

```diff
@@ -12,7 +13,7 @@ class Device {
       // TODO(erickzanardo): We probably could use dart:html and implement this for web as well
       return Future.value();
     }
-    return SystemChrome.setEnabledSystemUIOverlays([]);
+    return SystemChrome.setEnabledSystemUIMode(SystemUiMode.immersiveSticky);
```

Comment:
`immersiveSticky` 🤔  hahahahaha

Reply:
Btw, isn't this supported on web now? Maybe we could check and remove the if above (can be a follow up too)